routes/orders.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const ordersRouter = express.Router();
+const { getAllOrders, getCartByUser, createOrder } = require('../db');
+const { requireUser } = require('./utils');
+
+ordersRouter.use((req, res, next) => {
+    console.log("REQUEST is being made to /orders");
+    next();
+});
+
+
+ordersRouter.get('/', async (req, res, next) => {
```

Comment:
I like to see comments above each route, so that you can reference the code and know exactly which route this is.
`GET /api/orders/`